packages/elements/src/password-field/index.ts

```diff
@@ -17,9 +17,9 @@ let isEyeOffPreloadRequested = false;
 /**
  * A form control element for password.
  *
- * @fires value-changed - Dispatched when value changes
- * @fires error-changed - Dispatched when error state changes
- * @fires icon-click - Dispatched when icon is clicked
+ * @fires value-changed - Fired when user commits a value change. The event is not triggered if `value` property is changed by programmatically.
```

Comment:
20 & 21: 
* Fire when "the" user...
* property is changed ~~by~~ programmatically.